tutorial.md

```diff
@@ -344,14 +352,13 @@ Annotations:
 
 ### Create a Redis container inside the Pod
 
-Use the `crictl` command to pull the redis image, create a redis container from a container configuration and attach it to the Pod created earlier:
+Use the `crictl` command to pull the redis image, create a redis container from a container configuration and attach it to the Pod created earlier while capturing the container ID:
 
 ```
 sudo crictl pull redis:alpine
```

Comment:
Maybe change this to quay.io/crio/redis:alpine

Reply:
That would also make it so the change in container_redis.json is not necessary, I believe.

Reply:
Yes I hope so

Reply:
Confirmed, it does. Reverting the change in container_redis.json and including this change here.